src/utils/position.js

```diff
@@ -0,0 +1,6 @@
+import { subtract } from 'ramda';
+
+export const subtractPositions = (positionA, positionB) => ({
+  x: subtract(positionA.x, positionB.x),
```

Comment:
Any reason not to use the `-` operator here?

Reply:
Not really :D